Tests/SmithyHTTPAPITests/HeadersTests.swift

```diff
@@ -123,4 +124,21 @@ class HttpHeadersTests: XCTestCase {
         XCTAssertEqual(headerA, headerB)
         XCTAssertEqual(headerA.hashValue, headerB.hashValue)
     }
+
+    // MARK: - exists()
+
+    func test_exists_trueIfHeaderExists() {
+        let subject = Headers(["a": "abc"])
+        XCTAssertTrue(subject.exists(name: "a"))
+    }
+
+    func test_exists_falseIfHeaderDoesntExist() {
+        let subject = Headers(["a": "abc"])
+        XCTAssertFalse(subject.exists(name: "b"))
+    }
+
+    func test_exists_trueIfHeaderValueIsEmptyString() {
+        let subject = Headers(["a": ""])
+        XCTAssertTrue(subject.exists(name: "a"))
+    }
```

Comment:
These new tests check the expected behavior for the `exists()` method on `Headers`.